identity-and-access-management/secret-manager/quickstart.mdx

```diff
@@ -20,93 +20,42 @@ Scaleway’s [Secret Manager](/identity-and-access-management/secret-manager/con
-2. Click **Create secret**.
-3. Configure your secret:
-    * Select the region in which you want to create your secret. Your secret will be duplicated in all the Availability Zones of the selected region.
-    * Choose the [path](/identity-and-access-management/secret-manager/concepts/#path) in which you want to store your secret, or create a new path.
-    * Enter a name for your secret.
-    * Optionally, add tags.
-    * Add your secret manually or click <Icon name="plus" /> to import it from a file.
+1. Click **Secret Manager** in the **Security and Identity** section of the [Scaleway console](https://console.scaleway.com/) side menu.
+2. Click **+ Create secret**.
+3. Choose the [region](/identity-and-access-management/secret-manager/concepts/#region) in which you want to store your folder and your secret.
+    <Message type="important">
+     Folders and secrets cannot be moved from one region to another after creation.
+    </Message>
+4. Select **Create a new path** and enter `/`, followed by a name for your [folder](/identity-and-access-management/secret-manager/concepts/#folder).
+    <Message type="important">
+      A [path](/identity-and-access-management/secret-manager/concepts/#path) is the directory structure to access your folders and secrets. Each path **must be prefixed** with a slash.
+    </Message>
+5. Enter a name for your secret and add tags (optional).
+6. Add your secret manually or import it from a file.
     <Message type="note">
      The maximum file size for your secret is 64 KiB.
     </Message>
-4. Click **Create secret**.
-
-## How to manage a secret
-
-1. Click **Secret Manager** in the **Security and Identity** section of the [Scaleway console](https://console.scaleway.com/) side menu. Your newly created secrets display.
-2. Click <Icon name="more" /> next to the secret you want to manage.
-3. Click **More info**. Your secret's overview page displays.
-4. Click **Copy** to copy your secret's ID.
-5. Click the **Tags** bar and add more tags to organize your secret.
-6. Copy the code examples for the applications from which you want to get your secret.
-
-## How to create a version of a secret
-
-1. Click **Secret Manager** in the **Security and Identity** section of the [Scaleway console](https://console.scaleway.com/) side menu. Your secrets display.
-2. Click the secret you want to add a version to. Your secret's overview page displays.
-3. Click the **Versions** tab.
-4. Click <Icon name="plus" /> and **Create version**. A pop-up displays.
-5. Add your version manually or click <Icon name="plus" /> to import it from a file.
-6. Click <Icon name="toggle" /> to enable or disable the version.
-7. Click **Add new version**.
+6. Click **Create secret**. The **Overview** tab of your secret displays. You can copy your secret's ID, add a description, tags or delete your secret.
```

Comment:
I would add a message here saying clearly that:
- A secret has been created
- If the path contained folder that did not exist yet, all of them are created on the go
- the value of the secret is stored in the first version of this secret, which is enabled by default